vector/src/main/java/im/vector/app/features/start/StartAppViewEvent.kt

```diff
@@ -20,7 +20,7 @@ import im.vector.app.core.platform.VectorViewEvents
 
 sealed interface StartAppViewEvent : VectorViewEvents {
     /**
```

Comment:
<!--
  0 failure: 
  1 warning:  The first sentenc...
  
  
  DangerID: danger-id-Danger;
  File: vector/src/main/java/im/vector/app/features/start/StartAppViewEvent.kt;
  Line: 22;
-->

- :warning: The first sentence of this KDoc does not end with the correct punctuation.